app/src/main/java/com/shagalalab/constitution/ui/search/SearchResultFragment.kt

```diff
@@ -0,0 +1,83 @@
+package com.shagalalab.constitution.ui.search
+
+import android.os.Bundle
+import android.text.SpannableStringBuilder
+import android.view.View
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.Observer
+import androidx.navigation.NavController
+import androidx.navigation.Navigation
+import androidx.navigation.fragment.navArgs
+import androidx.recyclerview.widget.DividerItemDecoration
+import com.shagalalab.constitution.R
+import com.shagalalab.constitution.data.DataHolder
+import com.shagalalab.constitution.data.Language
+import com.shagalalab.constitution.data.models.ArticleModel
+import com.shagalalab.constitution.ui.article.ArticleViewModel
+import com.shagalalab.constitution.ui.search.adapter.ArticleItem
+import com.shagalalab.constitution.ui.search.adapter.LangItem
+import com.shagalalab.constitution.ui.search.adapter.ListItem
+import com.shagalalab.constitution.ui.search.adapter.ResultsAdapter
+import kotlinx.android.synthetic.main.fragment_search_result.*
+import org.koin.androidx.viewmodel.ext.android.viewModel
+
+class SearchResultFragment : Fragment(R.layout.fragment_search_result) {
+
+    private var adapter: ResultsAdapter = ResultsAdapter()
+    private val safeArgs: SearchResultFragmentArgs by navArgs()
+    private val viewModel: ArticleViewModel by viewModel()
+    private lateinit var navController: NavController
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        tvResults.text = getString(R.string.results_for) + "${safeArgs.query}\""
```

Comment:
There is no need for concatenating strings here, you can use [string formatting](https://developer.android.com/guide/topics/resources/string-resource#formatting-strings) instead